app/boards/intel_adsp_ace20_lnl.conf

```diff
@@ -40,6 +40,7 @@ CONFIG_MM_DRV_INTEL_ADSP_TLB_REMAP_UNUSED_RAM=y
 CONFIG_AMS=y
 CONFIG_COUNTER=y
 CONFIG_SOF_TELEMETRY=y
+CONFIG_SOF_TELEMETRY_PERFORMANCE_MEASUREMENTS=y
```

Comment:
with these changes performance measurements would be unconditionally enabled on MTL and LNL? Is this desirable?